Commands/Gambling/roulette.py

```diff
@@ -0,0 +1,72 @@
+# /commands/basic/help.py
+import discord
+from discord.ext import commands
+from typing import Literal
+import random as r
+
+# Define the slash command
+async def roulette_command(interaction: discord.Interaction, amount: int, color: Literal["🔴Red", "⚫Black"]):
+    """
+    Gambling Command. Selects a random number 1 - 38
+    if it is: {1, 3, 5, 7, 9, 12, 14, 16, 18, 19, 21, 23, 25, 27, 30, 32, 34, 36}, then it is Red
+    if it is: {2, 4, 6, 8, 10, 11, 13, 15, 17, 20, 22, 24, 26, 28, 29, 31, 33, 35}, then it is Black
+    if it is: {0, 37}, then it is Green
+
+    if it matches the pre-selected color, the user wins double what they invested
+    else, they lose everything
+
+    Green is a loss no matter what because you can only invest in Red/Black
+
+    Parameters: None
+    Returns: None
+    """
+    # variables
+    red_numbers = {1, 3, 5, 7, 9, 12, 14, 16, 18, 19, 21, 23, 25, 27, 30, 32, 34, 36}
+    black_numbers = {2, 4, 6, 8, 10, 11, 13, 15, 17, 20, 22, 24, 26, 28, 29, 31, 33, 35}
+    green_numbers = {0, 37}
+    rand_num = r.randint(0, 37)
+    col = discord.Color.blurple()
```

Comment:
Should probably name this as color, or something similar. `col` to me makes me think column at first. Not a huge deal though.

Reply:
updated